hedyorg/hedy
# 🧪 Improve tests

Fixes #3747 by also outputting the original code in the yaml (for easy searching) and by adding line numbers to code snippers 

**How to test**
Break a code snippet and observe that the output has changed, errors are now indicated with an arrow, and you get the original yaml too for easy searching

```
----
name is ask What is the name of the main character?
echo <-------
print name is now going to run in the woods
print name is a bit scared
print Suddenly he hears a crazy noise...
sleep
print name is afraid this is a haunted forest

----
name {is} {ask} What is the name of the main character?
{echo}
{print} name is now going to run in the woods
{print} name is a bit scared
{print} Suddenly he hears a crazy noise...
{sleep}
{print} name is afraid this is a haunted forest

----
```
